tailwind.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  purge: [],
```

Comment:
Please [enable the JIT mode](https://tailwindcss.com/docs/just-in-time-mode#enabling-jit-mode) in TailwindCSS, thanks!